app/views/games/index.html.erb

```diff
@@ -1,11 +1,30 @@
 <div class="booyah-box col-xs-10 col-xs-offset-1">
-<h1 class="text-center">List of Games</h1>
+<h3 class="text-center"><%= current_user.email %></h3>

```

Comment:
If the user is not signed in, this will result in a `undefined method `email' for nil:NilClass` error.  It could be under a conditional like `<% if current_user %>`, which would solve the problem.